project_generator/tools/iar.py

```diff
@@ -100,7 +100,7 @@ def _set_option(self, settings, value):
 
     def _get_option(self, settings, find_key):
         """ Return index for provided key """
-        # This is used as in IAR template, everything 
+        # This is used as in IAR template, everything
```

Comment:
I would prefer if this is separate commit , fine for now as the changes are easy to follow

Reply:
The whitespace changes you mean? Yeah, that's always a pain… Sorry I didn't separate them.